### PR TITLE
make WebDriver:findClickable() public, so that it can be used in Helpers

### PR DESCRIPTION
**Pull request by tiger-seo**

### CODE REVIEW

**Comment by DavertMik** [comment]
Please add documentation for `findClickable` similar to https://github.com/Codeception/Codeception/blob/2.2/src/Codeception/Lib/Interfaces/ElementLocator.php#L7 `findClickable` has signature different than `findElements` , it requires context element first so usage example should be added.

In the same manner please make `findCheckable` as public.

findClickable and findCheckable should not be added to ElementLocator interface, as they require WebDriver-only argument.

**Comment by DavertMik** [comment]
Doooooocumentation!

File: src/Codeception/Module/WebDriver.php [hunk 1]
```diff
@@ -821,11 +821,21 @@ public function click($link, $context = null)
...
+     * Locates clickable element.
+     *
+     * Use it in Helpers or GroupObject or Extension classes:
+     *
+     * ```php
+     * <?php
+     * $module = $this->getModule('WebDriver');
+     * $page = $module->webDriver;
+     * $el = $module->_findClickable($page, 'Click Me');
+     *
```
Comment (Naktibalda) [inline src/Codeception/Module/WebDriver.php]: You missed ``` here.

Comment (DavertMik) [inline src/Codeception/Module/WebDriver.php]: @tiger-seo please do better documentation next time

**Comment by DavertMik** [comment]
Ok, I will fix docs locally

**Pull request merged**